app/scripts/modules/core/src/healthCounts/HealthCounts.tsx

```diff
@@ -8,31 +8,31 @@ import { Tooltip } from 'core/presentation/Tooltip';
 
 import './healthCounts.less';
 
-export interface IHealthCountsPros {
+export interface IHealthCountsProps {
   container: IInstanceCounts;
   additionalLegendText?: string;
   legendPlacement?: Placement;
-}
+};
 
-export interface IHealthCountsState {
+interface IState {
```

Comment:
can you run `tsc` from the `core` directory and make sure this doesn't blow up? We have `declaration: true` set in the core package and I think this might be a problem.